README.md

```diff
@@ -361,6 +362,41 @@ const ColumnAdder = ({ addColumn }) {
 </Board>
 ```
 
+#### `renderCardAdder`
+
+Use this if you want to render your own card adder. You have to pass a function and return your card adder component.
+The function will receive these parameters:
+
+| Arg          | Description                                          |
+| -----------  | ---------------------------------------------------  |
+| `column`     | the column wich the card is being added              |
+
+| function     | Description                                                |
+| ------------ | ---------------------------------------------------------- |
+| `onConfirm`  | Call this function with a new card to add the new card |
+
+\* It's unavailable when the board is controlled.
+
+Ex.:
+
+```js
+const CardAdder = ({  }) {
+  return (
+    <div onClick={()=> onConfirm(column, {id: ${required-new-unique-columnId}, title: 'Title', description: 'description' })}>
+      Add column
+    </div>
+  )
+}
+
+<Board
+  allowAddCard
+  onCardNew={()=> console.log('Card added')}
+  onNewCardConfirm={(card) => ({ ...card, id: ${required-new-unique-cardId} })}
+  renderCardAdder={({ column,onConfirm }) => <CardAdder column={column} onConfirm={onConfirm} />}
+  {board}
+</Board>
```

Comment:
I think we should make this example more "workable".

Reply:
Yeah, updated it